prody/proteins/pdbfile.py

```diff
@@ -1086,6 +1093,9 @@ def writePDBStream(stream, atoms, csets=None, **kwargs):
     if resnums is None:
         resnums = np.ones(n_atoms, int)
 
+    serials = atoms._getSerials()
+    if serials is None or renumber == True:
```

Comment:
`if serials is None or renumber` is good. No need to compare `renumber` with `True`. It may give other developers the impression that `renumber` can be other value types.

Reply:
done